flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -149,6 +152,13 @@
 		.key(RocksDBProperty.ActualDelayedWriteRate.getConfigKey())
 		.defaultValue(false)
 		.withDescription("Monitor the current actual delayed write rate. 0 means no delay.");
+
+	public static final ConfigOption<Boolean> COLUMN_FAMILY_AS_VARIABLE = ConfigOptions
+		.key(METRICS_COLUMN_FAMILY_AS_VARIABLE_KEY)
+		.defaultValue(false)
+		.withDescription(String.format("Whether to expose the column family as a variable. " +
+			"The column-family-as-variable has default as '%s'", false));
```

Comment:
we don't need the default in the description; the tables in the documentation automatically list them.